models/experimental/functional_unet/tests/test_unet_model.py

```diff
@@ -65,9 +65,9 @@ def run_unet_model(batch, groups, device, iterations=1):
 @pytest.mark.parametrize("groups", [4])
 @pytest.mark.parametrize("device_params", [{"l1_small_size": UNET_L1_SMALL_REGION_SIZE}], indirect=True)
 def test_unet_model(batch, groups, device, use_program_cache, reset_seeds):
-    if (
-        not is_wormhole_b0(device)
-        and device.compute_with_storage_grid_size().x * device.compute_with_storage_grid_size().y != 110
+    if not is_wormhole_b0(device) and (
+        device.compute_with_storage_grid_size().x * device.compute_with_storage_grid_size().y != 110
```

Comment:
you should change the skip message below right?

Reply:
seems like we have this code for checking the "right" device in 3 places, maybe we can move it to a single place